projects/tad/index.md

```diff
@@ -0,0 +1,84 @@
+<!-- markdownlint-disable MD013 -->
```

Comment:
Maybe I missed the exact reason why we added this statement, but in the file `.markdownlint.yml` we set that MD013 should not be applied to tables. I think we could remove this markdownlint-disable statement here. What do you think?

Reply:
You are correct, this is no (longer) needed.